frontend/src/api/opsAuthAPI.js

```diff
@@ -27,8 +28,48 @@ export const opsAuthApi = createApi({
         getRoles: builder.query({
             query: () => `/roles/`,
             providesTags: ["Roles"]
+        }),
+        login: builder.mutation({
+            query: ({ provider, code }) => ({
+                url: "/login/",
+                method: "POST",
+                body: { provider, code }
+            }),
+            invalidatesTags: ["Auth"]
+        }),
+        logout: builder.mutation({
+            query: () => ({
+                url: "/logout/",
+                method: "POST"
+            }),
+            async onQueryStarted(_, { dispatch, queryFulfilled }) {
+                try {
+                    await queryFulfilled;
+                    // Dispatch logout action to clear auth state
+                    dispatch(logout());
+                } catch (err) {
+                    console.error("Error during logout:", err);
+                }
+            }
+        }),
+        refreshToken: builder.mutation({
```

Comment:
Where is this being invoked from now?

Reply:
invocations are in the `MultiAuthSection` like before but now it has been refactored to use RTK Query.

Reply:
I'm not seeing it in there for some reason, i.e. where `useRefreshTokenMutation` is used?

Reply:
added the refresh token in the `AuthSection` and tested locally and seems to work. Logged in and sat idle for a few minutes and logs out and redirects to login page.